cmd/kots/cli/admin-console-push-images.go

```diff
@@ -40,11 +40,6 @@ func AdminPushImagesCmd() *cobra.Command {
 			airgapArchive := args[0]
 			endpoint := args[1]
 
-			endpointParts := strings.Split(endpoint, "/")
-			if len(endpointParts) != 2 {
```

Comment:
i think we should still check that `endpointParts` is greater than 1, otherwise fail?

Reply:
allowing 0 or more parts now.